### ISSUE TITLE
KIF can't find elements in AddressBookUI

### ISSUE THREAD

**Issue by albertbori** [kif-framework/KIF#571]
Is Apple's address book ui inaccessible to KIF?

    tester.tapViewWithAccessibilityLabel("Add Contact")
    tester.tapViewWithAccessibilityLabel("Lasty")

> file:///Users/me/dev/myapp-ios/MyApp/AutomationTests/KIFExtensions.swift: test failure: -[MyTests testContacts()] failed: The step timed out after 10.00 seconds: Failed to find accessibility element with the label "Lasty"

![ios simulator screen shot feb 12 2015 11 35 15 pm](https://cloud.githubusercontent.com/assets/1742709/6191320/b5b57296-b365-11e4-942e-21fd53850820.png)

**Comment by phatmann** [comment]
I could not find anything definitive, but it does appear that the Address Book UI is running outside of the app and is therefore not accessible to KIF. Sorry :-(
